README.md

```diff
@@ -1,58 +1,42 @@
 
 ## Zeppelin API client plugin for Drone
```

Comment:
Should we use `CI/CD workflow` instead of `Drone` ?

Reply:
I think steadier is right about `CI/CD workflow`